### ISSUE TITLE
Config Request - Stop portals from generating when you fall from the Aether

### ISSUE THREAD

**Issue by GrinningMoon** [gildedgames/aether-issues#384]
Please add a config option so I can turn off portal generation whenever someone falls from the Aether. I'd only like the original player-made portal and the paired portal that generates in the Aether to generate. Please add an option to prevent portals from generating when a player falls out of the Aether. This seems like it could be easily abused for glowstone if the player has some sort of slowfall, or it can be messy with portals generating everywhere a player falls to their death.

**Comment by CurePindal** [comment]
This should rather be labeled as "bug", because of how its exploitable!
Very annoying bug, btw!

**Comment by aridhol** [comment]
Yeah, I'd second this also because its exploitable.. I'm perfectly fine with people being stranded upon falling, its their own fault...

**Comment by xJonL** [comment]
@CurePindal It's not a bug, it's the way we've designed our own portal system. This is more of a design issue.

**Comment by Collin1971** [comment]
We've been looking into the problem for a little bit but unfortunately we weren't able to release any kind of fix in the 1.7.10-1.1 release. Our goal is to find a better way to go about it before we release the next update.

**Comment by Collin1971** [comment]
This will most likely be removed instead of giving an config option.

**Comment by Collin1971** [comment]
We believe we've fixed the issue and are doing some testing to make sure it's actually fixed. :)

**Comment by GrinningMoon** [comment]
Thank you for the updates! I did some brief testing on the updated versions and it seems that the portals don't spawn anymore, which is great! However, I did notice that Aerbunnies do still cause portals to generate. I thought I remember reading that the Aerbunny part couldn't be fixed in 1.7.10. Can you confirm this? I'd like for it to be fixed for 1.7.10 as it does cause a clutter but if not, I'm sure there's some sort of workaround. Thank you again for keeping on top of this!

**Comment by babybasher** [comment]
Surely this can be fixed in 1.7.10!

Like the above posted I don't want portals appearing everywhere in my world.

From a coding perspective I find this very very odd that it apparently can't be fixed.

**Comment by Collin1971** [comment]
The issue came from us not being able to find out _how_ the player fell out of the world and how to handle it. We eventually fixed it (with kind of a hacky fix of just _not_ creating a portal if the source was the Aether) but it took a lot of testing to actually make sure we didn't break anything else.

It's fixed in 1.7.10-1.3 for sure, though from memory right now I can't remember if we fixed it in 1.7.10-1.3. Sorry for not updating this post.
